Funky-Chickens/workFORCEmanagement
# Gm migrations

### I made the following changes:

- fix config file and add to gitignore, fix spelling errors.
- add migration files for join tables with full schemas, 
- modify training model to be uniform with other foreign key constraints, 
- change allow null to true in other migration files where necessary
- fixed employee model to reflect id needed by migrations for join tables, (Thanks, Em!)
- deleted trailing spaces, 
- finished migrating all files to test

### Reason:
- to create join tables for Employee Computers and Employee Trainings
- to test order of migrations with the join tables included

### To Test:
- run ```sequelize db:migrate```
- run ```psql``` in a new terminal window
- enter ```\c bangazon_workforce```  to connect to the database on your local machine
- run ```\d ``` to see that the tables exist in the database 

### Expected Outcome:
- The list of tables should include all migrated tables.
